fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -80,7 +117,15 @@ public static void dropDatabase(String database) {
         if (!database.contains(DERBY_DIR)) {
             throw new IllegalArgumentException("Derby databases must start with: " + DERBY_DIR);
         }
-
+        
+        // Make sure the database is shut down before we try to drop it
+        try {
+            shutdown(database);
+        }
+        catch (IllegalStateException x) {
+            // NOP - database doesn't exist anyway
```

Comment:
please add a logger statement here

Reply:
added logger.info("DROP DATABASE - database does not exist: " + database);